docs/Reference/Configuration-File.md

```diff
@@ -71,6 +79,43 @@ Details of the operator's engine component.
 
 Address (host and port) of the engine.
 
+### `ha`
+
+Configure ConsenSys Rollups to be [highly available](../Concepts/High-Availability.md).
+
+### `ha.kafka`
+
+Configure the [highly available](../Concepts/High-Availability.md) Kafka settings.
+
+#### `brokers`
+
+Address of the highly available [Kafka brokers].
+
+#### `pending_operations_replication_topic`
```

Comment:
we should mention that each operator instance **must** write to a different topic, e.g 

- instance 0 writes to `operatorX-instance0-invalid-operations`;
- instance 1 writes to `operatorX-instance1-invalid-operations`;
- ...

`consumer_group_name` Ideally, different for each operator instance as well.